documentation/source/ModelPredictions.md

```diff
@@ -0,0 +1,181 @@
+# Using Pretrained Models for Predictions 
+
+In this tutorial, we will demonstrate how to use the `model.predict()` method for object detection tasks.
+
+The model used in this tutorial is [YOLO-NAS](YoloNASQuickstart.md), pre-trained on the [COCO dataset](https://cocodataset.org/#home), which contains 80 object categories.
+
+
+**Warning**: If your model was trained on your own custom dataset, make sure to follow these [steps first](DetectionPredictionCustom.md).
```

Comment:
Its a bit unclear...
A custom dataaset can also be one that uses our COCO Format Detection Dataset class.
In that case (and actually in any case where we just trian with SG objects and use our transforms for detection) there is nothing to be done (recall my feature that takes the validation transforms and creates a pipeline from it).

Please also mention that, as in that case you save the users trouble defining these things.

Reply:
35af53371555093a4ab5aea8bb73120296bb8ee6
I refered to SG datasets, and not transforms explicitly because the SG detection datasets all work with SG transforms, so I think there is no point to go too deep in implementations details:
What I wrote;
You are using SG dataset ? -> No need to set the params
You are not ? -> set the params